### ISSUE TITLE
[Quest][Northrend] Post-Partum Aggression - Missing questcredit

### ISSUE THREAD

**Issue by cjwyatt** [dalaranwow/dalaran-wow#5075]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
After accepting the quest I am auto-mounted into the mammoth, Stampy. I then ride to the Broodmother Slivinia and engage. After a short while I kill the mob. Hemet Nesinwary can be seen attacking her too. Once the mob dies Hemet despawns and the quest is not marked as completed.
**Expected behaviour**: Tell us what should happen instead
After the mob dies Dorian from the camp should show up and have a conversation with Hemet and then the quest should be marked complete.
**Steps to reproduce the problem**:

1. Accept quest
2. Kill Broodmother Slivinia whilst mounted on the mammoth

**Include proofs for this behaviour**
What kind of proof are you looking for? If you want proof of how it should behave see http://www.wowhead.com/quest=12614/post-partum-aggression

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Broodmother Slivinia - 28467

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by cjwyatt** [comment]
Just tried this again and it's still broken.
